vote/src/vote_state_view/frame_v1_14_11.rs

```diff
@@ -0,0 +1,109 @@
+use {
+    super::{
+        field_frames::{AuthorizedVotersListFrame, ListFrame, PriorVotersFrame, RootSlotFrame},
+        EpochCreditsListFrame, Field, Result, VoteStateViewError, VotesListFrame,
+    },
+    solana_pubkey::Pubkey,
+    solana_vote_interface::state::BlockTimestamp,
+    std::io::BufRead,
+};
+
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
+pub(super) struct VoteStateFrameV1_14_11 {
+    pub(super) num_votes: u8,
+    pub(super) has_root_slot: bool,
+    pub(super) num_authorized_voters: u8,
+    pub(super) num_epoch_credits: u8,
+}
+
+impl VoteStateFrameV1_14_11 {
+    pub(super) fn try_new(bytes: &[u8]) -> Result<Self> {
+        let votes_offset = Self::votes_offset();
+        let mut cursor = std::io::Cursor::new(bytes);
+        cursor.set_position(votes_offset as u64);
+
+        let votes = VotesListFrame::read(&mut cursor, false /* has_latency */)?;
+        let root_slot = RootSlotFrame::read(&mut cursor)?;
+        let authorized_voters = AuthorizedVotersListFrame::read(&mut cursor)?;
+        PriorVotersFrame::read(&mut cursor);
+        let epoch_credits = EpochCreditsListFrame::read(&mut cursor)?;
+        cursor.consume(core::mem::size_of::<BlockTimestamp>());
+        if cursor.position() as usize <= bytes.len() {
```

Comment:
trailing bytes are allowed. consistent with default behavior of **function** `bincode::deserialize`

Reply:
added a comment for this